backend/taxHandlers.ts

```diff
@@ -8,13 +8,70 @@ import {
   FetchTaxesPayload,
   TaxJarConfig,
   OrderPayload,
+  LinePayload,
+  DiscountPayload,
+  FetchTaxesLinePayload,
 } from "./types";
 
+const getDiscountForLine = (
+  line: LinePayload,
+  totalDiscount: number,
+  allLinesTotal: number
+) => {
+  if (totalDiscount === 0 || allLinesTotal == 0) {
```

Comment:
```suggestion
  if (totalDiscount === 0 || allLinesTotal === 0) {
```

Reply:
JS coertion rules are too much for human beings - you can consider adding this rule to ESLint: https://eslint.org/docs/rules/eqeqeq